scripts/ci/test_pytorch_blade.sh

```diff
@@ -36,7 +40,7 @@ fi
 export TORCH_BLADE_CI_BUILD_TORCH_VERSION=${TORCH_BLADE_CI_BUILD_TORCH_VERSION:-1.7.1+cu110}
 (cd pytorch_blade && bazel clean --expunge \
   && python -m pip install -q -r requirements-dev-${TORCH_BLADE_CI_BUILD_TORCH_VERSION}.txt \
-       -f https://pai-blade.oss-cn-zhangjiakou.aliyuncs.com/pytorch/wheels/repo.html \
+       -f https://download.pytorch.org/whl/torch_stable.html \
```

Comment:
PyTorch official web server would reject our CI if downloading too frequently, cc @fortianyou

Reply:
I noticed that we already use the official web server [here](https://github.com/alibaba/BladeDISC/blob/main/pytorch_blade/ci_build/build_pytorch_blade.sh#L34). So, I expect this would not be a big problem? Furthermore, the wheel file will be cached in each CI machine.

Reply:
@wyzero I added the official index-url [here](https://github.com/alibaba/BladeDISC/blob/main/pytorch_blade/ci_build/build_pytorch_blade.sh#L34) because I wanted to remind the user to get an officially released PyTorch version.

The [scripts/ci/test_pytorch_blade.sh](https://github.com/alibaba/BladeDISC/pull/240/files/f907ea9222d415b03279a847b88274e88c796717#diff-68dbc64436f796474ead988939c8925fa666aa145fb5a5f524b0d6f2669ea7de) will run firstly and the PyTorch installation would be cache, so we will run out of downloading from https://download.pytorch.org/whl/torch_stable.html.

Reply:
I think the disc cache is enough? we do not need to download the  wheel package each time.  Furthermore, some torch versions (e.g. for arm) are not in our customized server.

Reply:
I agree that we can preinstall PyTorch. But it seems we haven't done this. `scripts/ci/test_pytorch_blade.sh` will be run every time when CI. cc @wyzero @Yancey1989